tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs

```diff
@@ -114,9 +115,38 @@ public void ApplicationOptionsIncludeClientSecret(string clientSecret)
             }
         }
 
+        [Theory]
+        [InlineData(null, false)]
+        [InlineData("", false)]
+        [InlineData("notAUri", false)]
+        [InlineData("htt://nonhttp/", false)]
+        [InlineData("https://login.microsoftonline.com/", true)]
+        [InlineData("https://login.microsoftonline.com", true)]
+        [InlineData("https://cats.b2clogin.com/", true)]
+        [InlineData("https://cats.b2clogin.com/signout-callback-oidc", true)]
+        [InlineData("http://cats.b2clogin.com/signout-callback-oidc", true)]
+        public void ValidateRedirectUriFromMicrosoftIdentityOptions(
+            string redirectUri,
+            bool isTrue)
+        {
+            string httpContextRedirectUri = "https://IdentityDotNetSDKAutomation/";       
```

Comment:
How does this value make into _tokenAcquisition ?

Reply:
through the MicrosoftIdentityOptions, which are injected by DI in the TokenAcquisition constructor